<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">

    <!-- Latest compiled and minified CSS -->
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">

    <!-- jQuery library -->
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>

    <!-- Latest compiled JavaScript -->
    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/js/bootstrap.min.js"></script>

    <link rel="stylesheet" type="text/css" href="css/mystyle.css">
            
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">    
    
    <title>Research Profile</title>    
    <link rel="shortcut icon" href="images/logo.ico" type="image/x-icon" />
</head>

<body>
   <div class="wrapper mnWidth500px">

        <nav class="navbar navbar-inverse navbar-fixed-top">
            <div class="container-fluid">
                <div class="navbar-header">
                    <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#myNavbar">
                            <span class="icon-bar"></span>
                            <span class="icon-bar"></span>
                            <span class="icon-bar"></span>                        
                    </button>
                    <a class="navbar-brand" href="index.html">Yo, Tahsin Masrur here!</a>
                </div>
                <div class="collapse navbar-collapse" id="myNavbar">
                    <ul class="nav navbar-nav">
                    </ul>
                    <ul class="nav navbar-nav navbar-right">
                        <li><a href="index.html">Home</a></li>
                        <li><a href="contests.html">Programming Contest Profile</a></li>
                        <li><a href="problems.html">Programming Contest Problems I Set</a></li>
                        <li class="active"><a href="research.html">Research Profile</a></li>
                        <li><a href="career.html">Career Background</a></li>
                    </ul>
                </div>
            </div>
        </nav>

       <div class="belowNav"></div>

        <div class="container">
            <p align="justify">
                <br/>
                For research purposes, I focused on machine learning and tried to gain as much knowledge as possible on linear algebra and statistics to build up my fundametals. Eventually, I learned a lot of machine learning/data mining techniques like classification, clustering, feature selection and feature extraction methods. Some of them are <i>SVM, Student's t-test, PCA, Kruskal-Wallis test, Mann-Whitney-Wilcoxon test, KNN, decistion trees, mRMR</i> etc. As evident from my publications, I applied my knowledge on bioinformatics and worked specially on the impact of feature selection methods in this sector. 
            </p>
            
            <h3>PUBLICATIONS</h3>
            <hr>
            <div class="panel-group">
              <div class="panel panel-primary">
                <div class="panel-heading">Paper Title: <b>Metabolomic Biomarker Identification for Lung Cancer By Combining Multiple Statistical Approaches</b></div>
                <table class="table table-striped table-borderless" align="center">
                    <tbody>
                        <tr>
                            <td><b>Authors</b></td>
                            <td>Tahsin Masrur, Md. Al Mehedi Hasan, Md. Nazrul Islam Mondal</td>
                        </tr>
                        <tr>
                            <td><b>Conference</b></td>
                            <td>2019 International Conference on Electrical, Computer and Communication Engineering (ECCE), 7-9 February, 2019</td>
                        </tr>
                        <tr>
                            <td><b>Conference Location</b></td>
                            <td>Cox's Bazar, Bangladesh</td>
                        </tr>
                        <tr>
                            <td><b>Organized By</b></td>
                            <td>Faculty of Electrical and Computer Engineering, Chittagong University of Engineering &amp; Technology, Bangladesh</td>
                        </tr>
                        <tr>
                            <td><b>Publisher</b></td>
                            <td>IEEE</td>
                        </tr>
                        <tr>
                            <td><b>DOI</b></td>
                            <td>10.1109/ECACE.2019.8679222</td>
                        </tr>
                        <tr>
                            <td><b>URL</b></td>
                            <td><a href="https://ieeexplore.ieee.org/document/8679222" target="_blank">https://ieeexplore.ieee.org/document/8679222</a></td>
                        </tr>
                        <tr>
                            <td><b>Abstract</b></td>
                            <td align="justify">
                                Metabolomic biomarkers are tools that can be used in early disease prediction and drug designing for diseases like lung cancer. Knowing the most differentially expressed metabolites creates a much higher probability of diagnosing lung cancer faster than normal, which can reduce the mortality rate. They are crucial during drug design too. Previously, various works have been done on discovering biomarkers for different diseases. However, it is still nowhere near sufficient since reducing the number of biomarkers and maintaining good classification accuracy are urgent issues in a sector where people's lives are at stake. Thus, to contribute more, in this paper, we have identified the influential metabolites in plasma and serum blood sample for lung cancer and then selected biomarkers from them. We first considered a parametric test (Student's t-test) and two non-parametric tests (Kruskal-Wallis and Mann-Whitney-Wilcoxon test) to identify the influential metabolites. We also differentiated the up-regulated and down-regulated metabolites using FC values and heatmap plot. We used SVM classifier to ascertain good accuracy with our set of influential metabolites and ROC Curve Analysis to rank the metabolites and choose biomarkers. Our analysis resulted in 28 influential (p-value &lt; 0.05) metabolites from plasma sample and 13 influential (p-value &lt; 0.05) metabolites from serum sample. Finally, 10 metabolites were chosen from each of the samples as respective biomarkers. All the files and codes used in our work are available at <a href="https://github.com/Zeronfinity/LungCancerBiomarkers" target="_blank">https://github.com/Zeronfinity/LungCancerBiomarkers</a>.
                            </td>
                        </tr>
                    </tbody>
                </table>
              </div>
              <br/>
              <div class="panel panel-primary">
                <div class="panel-heading">Paper Title: <b>Identification of Metabolomic Biomarker using Multiple Statistical Techniques and Recursive Feature Elimination</b></div>
                <table class="table table-striped table-borderless" align="center">
                    <tbody>
                        <tr>
                            <td><b>Authors</b></td>
                            <td>Tahsin Masrur, Md. Al Mehedi Hasan</td>
                        </tr>
                        <tr>
                            <td><b>Conference</b></td>
                            <td>International Conference on Computer, Communication, Chemical, Materials and Electronic Engineering (IC4ME2), 11-12 July, 2019</td>
                        </tr>
                        <tr>
                            <td><b>Conference Location</b></td>
                            <td>University of Rajshahi, Rajshahi, Bangladesh</td>
                        </tr>
                        <tr>
                            <td><b>Organized By</b></td>
                            <td>Faculty of Engineering, University of Rajshahi, Bangladesh</td>
                        </tr>
                        <tr>
                            <td><b>Publisher</b></td>
                            <td>IEEE</td>
                        </tr>
                        <tr>
                            <td><b>DOI</b></td>
                            <td>Still on process</td>
                        </tr>
                        <tr>
                            <td><b>URL</b></td>
                            <td>Still on process</td>
                        </tr>
                        <tr>
                            <td><b>Abstract</b></td>
                            <td align="justify">
                                Mortality rate of diseases like lung cancer can be decreased significantly by increasing the chance of early diagnosis. Identifying differentially expressed (DE) metabolites may contribute remarkably in this concern, and also in drug design. In the past, several kinds of approaches were attempted to discover biomarkers for diseases. Nonetheless, discovering compact-sized biomarkers while maintaining satisfactory classification performance is still a challenge. Therefore, for further contribution in this sector, we have declared biomarkers from our identified DE metabolites in plasma and serum blood sample of lung cancer. Student's t-test, Kruskal-Wallis and Mann-Whitney-Wilcoxon test were applied to distinguish the DE metabolites. Cluster heatmap plot and fold change values were used to differentiate between up and down-regulated metabolites. Finally, RFE method was used to order the metabolites and select biomarkers from them. To assess the performance with our DE metabolites or biomarkers, SVM classifier was utilized. We found 28 DE metabolites from plasma dataset and 13 from serum (p-value&lt;0.05). In the end, 8 metabolites were selected from plasma sample and 5 were selected from serum sample as the metabolomic biomarkers. The relevant files and codes of our work can be found at <a href="https://github.com/Zeronfinity/LungCancerBiomarkerRFE">https://github.com/Zeronfinity/LungCancerBiomarkerRFE</a>.
                            </td>
                        </tr>
                    </tbody>
                </table>
              </div>
              <br/>
              <div class="panel panel-primary">
                <div class="panel-heading">Paper Title: <b>Predicting N<sup>1</sup>- and N<sup>6</sup>-methyladenosine RNA Modifications using Hybrid Feature Selection Approach</b></div>
                <table class="table table-striped table-borderless" align="center">
                    <tbody>
                        <tr>
                            <td><b>Authors</b></td>
                            <td>Tasfin Jayed, Md. Al Mehedi Hasan, Tahsin Masrur (also presenter)</td>
                        </tr>
                        <tr>
                            <td><b>Received</b></td>
                            <td>Dr. Fatema Rashid Best Paper Award</td>
                        </tr>
                        <tr>
                            <td><b>Conference</b></td>
                            <td>International Conference on Advances in Electrical Engineering (ICAEE), 26-28 September, 2019</td>
                        </tr>
                        <tr>
                            <td><b>Conference Location</b></td>
                            <td>Independent University, Dhaka, Bangladesh</td>
                        </tr>
                        <tr>
                            <td><b>Organized By</b></td>
                            <td>Department of Electrical and Electronic Engineering, Independent University, Bangladesh (IUB)</td>
                        </tr>
                        <tr>
                            <td><b>Publisher</b></td>
                            <td>IEEE</td>
                        </tr>
                        <tr>
                            <td><b>DOI</b></td>
                            <td>10.1109/ICAEE48663.2019.8975621</td>
                        </tr>
                        <tr>
                            <td><b>URL</b></td>
                            <td><a href="https://ieeexplore.ieee.org/document/8975621" target="_blank">https://ieeexplore.ieee.org/document/8975621</a></td>
                        </tr>
                        <tr>
                            <td><b>Abstract</b></td>
                            <td align="justify">
                                RNA modification refers to the local structural changes or new chemical group additions in nucleotides. It has impact on some crucial biological activities and is also linked to several serious diseases, e.g. leukemia, breast cancer, zika virus and so on. Thats why the identification of RNA modifications attains great concern. N1-methyladenosine (m1A) and N6-methyladenosine (m6A) are two frequent modifications which occur at the adenosine site of RNA. So far, various methods have been developed to predict the modifications, e.g. iRNA-3typeA, RAM-ESVM etc. However, these methods can be improved further with the help of multiple feature selection approaches. In this paper, we have done extensive analysis on the effect of multiple feature selection methods and proposed a hybrid feature selection approach. This hybrid feature selection approach considers the common features that have been selected by Students t-test, Kruskal-Wallis test and minimum redundancy maximum relevance (mRMR) method. Applying this approach with 10-fold cross-validation and support vector machine classifier, we have obtained 99.37% and 91.02% accuracy for m1A and m6A (Homo sapiens), and 89.97% and 98.17% accuracy for m1A and m6A (Mus musculus) respectively.
                            </td>
                        </tr>
                    </tbody>
                </table>
              </div>
            </div>
        
        </div>

        <div class="push"></div>
       
    </div>

    <footer class="container-fluid text-center">
        <p>
            <a href="mailto:tahsin86m@gmail.com" class="btn btn-social-icon btn-email"><span class="fa fa-envelope"> Email</span></a>
            <a target="_blank" href="https://github.com/Zeronfinity" class="btn btn-social-icon btn-github"><span class="fa fa-github"> GitHub</span></a>
            <a target="_blank" href="https://www.linkedin.com/in/tahsin86m/" class="btn btn-social-icon btn-linkedin"><span class="fa fa-linkedin"> LinkedIn</span></a>
            <a target="_blank" href="https://www.facebook.com/anknown.miistt" class="btn btn-social-icon btn-facebook"><span class="fa fa-facebook"> Facebook</span></a>
            <br/>&#9400; 2019 zeronfinity.github.io | By Tahsin Masrur
        </p>
    </footer>
       
</body>

</html>